jekyll/_cci2/collect-test-data.md

```diff
@@ -1,47 +1,63 @@
 ---
 layout: classic-docs
-title: "Collecting Test Metadata"
-short-title: "Collecting Test Metadata"
-description: "Collecting test metadata"
-order: 34
+title: "Collecting test data"
+description: "A guide to collecting test data in your CircleCI projects."
 version:
 - Cloud
 - Server v3.x
 - Server v2.x
 ---
 
-CircleCI collects test metadata from XML files and uses it to provide insights into your job. This document describes how to configure CircleCI to output test metadata as XML for some common test runners and store reports with the `store_test_results` step.
+When you run tests in CircleCI there two ways to store your test results: using either artifacts or the {{site.url}}/{{site.baseurl}}/2.0/configuration-reference/#storetestresults[`store_test_results` step]. There are advantages to both methods, so the decision needs to be made for each project.
+
+When you save test data using the [`store_test_results` step]({{site.baseurl}}/2.0/configuration-reference/#storetestresults), CircleCI collects data from XML files and uses it to provide insights into your job. This document describes how to configure CircleCI to output test data as XML for some common test runners and store reports with the `store_test_results` step.
 
 * TOC
 {:toc}
 
+## Overview
+
+Using the **`store_test_results` step** gives you access to:
+
+* The **Tests** pane in the CircleCI web app, as described in the {{site.url}}/{{site.baseurl}}/2.0/collect-test-data/[Collecting Test Metadata guide].
+* Test insights and flaky test detection.
+* Test splitting
```

Comment:
I think we should be consistent in a bulleted list if we're using full stops or not after each item.

Reply:
Oh yeah ooops